client/lcd/test_helpers.go

```diff
@@ -135,30 +143,34 @@ func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress
 	genDoc, err := tmtypes.GenesisDocFromFile(genesisFile)
 	require.NoError(t, err)
 
-	if nValidators < 1 {
-		panic("InitializeTestLCD must use at least one validator")
+	// append initial (proposing) validator
+	genDoc.Validators[0] = tmtypes.GenesisValidator{
+		PubKey: privVal.GetPubKey(),
+		Power:  1,
```

Comment:
The reason this is stalling is because the one validator which is actually online aka "validator-1" only has a fraction of the voting power, the network will not stall if it's power is greater than 2/3, ideally should be 99.9999% of the voting power so it's the only proposer for all blocks, hence no down time... let's switch power to something much larger? maybe `999999`?

Reply:
Aight...This is enough for me to resolve the above TODO.